sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -419,7 +419,7 @@ case class CatalogTable(
     map ++= storage.toLinkedHashMap
     if (tracksPartitionsInCatalog) map.put("Partition Provider", "Catalog")
     if (partitionColumnNames.nonEmpty) map.put("Partition Columns", partitionColumns)
-    if (schema.nonEmpty) map.put("Schema", schema.treeString)
+    if (schema.nonEmpty) map.put("Schema", schema.simpleString)
```

Comment:
this may truncate, shall we use `catalogString`?

Reply:
OK